.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
:+1: I assume this is what we were missing for the issue picker.

Reply:
Yes. I think just the existence of the file will trigger the picker, but since we're here, let's also not allow blank issues :)